source/slang/slang-compiler.h

```diff
@@ -1277,9 +1421,24 @@ namespace Slang
     struct TypeCheckingCache;
     //
 
-    class Session
+    class Session : public RefObject, public slang::IGlobalSession
```

Comment:
Do we want to rename Session -> GlobalSession?

Reply:
Yeah, that would be helpful, but I think that needs to be a distinct refactoring/renaming pass because it will change so many source files that it could make the diffs confusing. If we want to do that rename as a separate PR *first*, I could do that.